habushu-mixology-consumer/tests/features/unpack-deps.feature

```diff
@@ -5,4 +5,8 @@ Feature: Verify that dependencies get unpacked correctly for tests
     When I reference a dependency in my python code
     Then the build can successfully execute the tests
 
```

Comment:
I (Informational): just adding a test to catch any future regression where transitive dependencies don't make it into the project.  Not a likely scenario, but at least there is a trip wire.